presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1288,6 +1271,30 @@ private void testInsertPartitionedBucketedTableWithUnionAll(HiveStorageFormat st
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertTwiceToSamePartitionedBucket()
+    {
+        String tableName = "test_duplicated_insert_with_partitioned_bucketed_table";
+        createPartitionedBucketedTable(tableName, HiveStorageFormat.RCBINARY);
```

Comment:
why not to enumerate all storage formats like we do for tests with insert into non partitioned bucketed table?

Reply:
@kokosing what's the value of running this test for various storage formats?

Reply:
I am not sure. It was a convention in this file. So if we are breaking convention then maybe let's change the approach in existing tests.